UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs

```diff
@@ -694,6 +694,87 @@ public void UpdateClientBodyPartStats(BodyPartType bodyPartType, float bruteDama
 		}
 	}
 
+	/// ---------------------------
+	/// Electrocution Methods
+	/// ---------------------------
+	/// Note: Electrocution for players is extended in PlayerHealth deriviative.
```

Comment:
So PH and LHB have different logic for electrocution...how can I test what happens when something that's just an LHB gets electrocuted? Seems like all I can do is electrocute players so I'd never be able to know if this implementation works.